frontend/components/panels/HighestBidCheckPanel.vue

```diff
@@ -78,30 +74,46 @@ export default Vue.extend({
             type: String,
             default: null,
         },
+        isExplanationsShown: {
+            type: Boolean,
+            default: true,
+        },
     },
     computed: {
         latestBid(): BigNumber | null {
             return this.auction.bidAmountMKR.isEqualTo(0) ? null : this.auction.bidAmountMKR;
         },
         isUserLatestBidder(): boolean {
-            return !!this.latestBid && this.auction.receiverAddress === this.userWalletAddress;
+            return !!this.latestBid && this.auction.receiverAddress === this.walletAddress;
         },
         currentStateAndTitle(): PanelProps {
-            if (!this.userWalletAddress && !this.latestBid) {
+            if (!this.walletAddress && !this.latestBid) {
                 return {
-                    name: 'inactive',
+                    name: 'notice',
                     title: `No bids yet`,
                 };
             }
+            if (this.auction.state === 'ready-for-collection') {
+                if (this.isUserLatestBidder) {
+                    return {
+                        name: 'inactive',
+                        title: `You are the highest bidder`,
+                    };
+                }
+                return {
+                    name: 'inactive',
+                    title: `You are not the highest bidder`,
+                };
+            }
```

Comment:
can we shorten this and 

```js
return { 
  name: 'inactive',
  title: this.isUserLatestBidder ? optionA : optionB
}
```
also for some reason the string is in ` instead of '